src/coreclr/vm/olevariant.cpp

```diff
@@ -4298,8 +4296,15 @@ void OleVariant::ConvertContentsBSTRToString(BSTR bstr, STRINGREF *pStringObj)
     if (hasTrailByte)
     {
         BYTE* buff = (BYTE*)bstr;
```

Comment:
Instead of introducing reverse FCalls for Get/SetTrailByte, can we just call BSTRMARSHALLER__CONVERT_TO_NATIVE/MANAGED?